technology-overview.md

```diff
@@ -0,0 +1,81 @@
+# Technology overview
+
+[Agoric](https://agoric.com/about/) aims at enabling safe cooperation among otherwise mutually suspicious parties
```

Comment:
Why "otherwise"? I think the sentence reads better and is more accurate with it removed.

Reply:
Removed in next commit